htmlbook-autogen/document.html.erb

```diff
@@ -7,7 +7,8 @@ xmlns="http://www.w3.org/1999/xhtml">
 <% toc_levels = (attr? :levels) ? (attr :levels).to_i : (@document.attr :toclevels, 2).to_i %>
   <head>
     <title><%= doctitle %></title>
-    <meta name="<%= doctitle %>" content="text/html; charset=utf-8"/>
+    <meta charset="utf-8"/>
+    <meta name="title" content="<%= doctitle %>"/>
```

Comment:
That's definitely an error.  Would a better fix be:
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> 

Is the html5 charset attribute supported in the epub target?